doc/pachub/pachub_getting_started.md

```diff
@@ -97,31 +94,7 @@ on Pachyderm Hub.
 
 To connect to your cluster, complete the following steps:
 
-1. On your local computer, open a terminal window.
-1. Install `pachctl` for your platform. For example, if you are using
-   macOS, run:
-
-   ```bash
-   $ brew tap pachyderm/tap && brew install pachyderm/tap/pachctl@1.9
-   ```
-
-   If you are using another operating system, see
-   [Install `pachctl`](../getting_started/local_installation.html#install-pachctl).
-
-   * If you already have `pachctl` installed, skip this step, or you
-   might need to update your version of `pachctl`. For example, if you use
-   macOS and `brew`, run:
-
-   ```bash
-   $ brew upgrade pachyderm/tap/pachctl@1.9
-   ```
-
-1. Verify your `pachctl` version:
-
-   ```bash
-   $ pachctl version --client-only
-   1.9.7
-   ```
+1. [Install or upgrade `pachctl`](http://docs.pachyderm.com/en/latest/getting_started/local_installation.html#install-pachctl).
```

Comment:
I figured this is better because the linked docs also includes instructions for installing on linux